machine/v1beta1/types_awsprovider.go

```diff
@@ -167,6 +167,13 @@ type EBSBlockDeviceSpec struct {
 	// it is not used in requests to create gp2, st1, sc1, or standard volumes.
 	// +optional
 	Iops *int64 `json:"iops,omitempty"`
+	// throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
+	//
+	// This parameter is valid only for gp3 volumes.
+	//
+	// Valid Range: Minimum value of 125. Maximum value of 1000.
+	// +optional
+	Throughput *int32 `json:"throughput,omitempty"`
```

Comment:
Just wondering if we need to mention the default here? That should be `125` MiB/s right?

Reply:
Also, it is only valid for gp3 volumes. I don't know what behavior would occur if we set it to 125 on a non-gp3 volume. 0 is a good default.

Reply:
Right, I was thinking of mentioning in the description about what the default value will be set if not configured.

But i guess if Iops is not saying anything, we can skip it :D

Reply:
@jhixson74 Though, one more question: I see the we define the valid range

> Valid Range: Minimum value of 125. Maximum value of 1000.

Do we expect this change in the future (i.e. AWS might announce new limits)? If so, should we follow the field `Iops` and say this instead:

```go
// Minimal and maximal throughput for gp3 are constrained. Please, check
// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
// for precise boundaries for individual volumes.
```

Reply:
We tend to include this kind of information, in a sentence approximately
```
When omitted, this means no opinion, and the platform is left to choose a reasonable default, which is subject to change over time.
The current default is 125.
```

Reply:
Fixed.